package/libs/toolchain/Makefile

```
@@ -490,6 +490,7 @@ ifeq ($(CONFIG_EXTERNAL_TOOLCHAIN),)
   define Package/libstdcpp/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(TOOLCHAIN_DIR)/lib/libstdc++.so.* $(1)/usr/lib/
+	rm -rf $(1)/usr/lib/*-gdb.py
```

Why not adapt the wildcard above accordingly? A real `.so` should always end with something numeric.
```suggestion
 	$(CP) $(TOOLCHAIN_DIR)/lib/libstdc++.so.*[0-9] $(1)/usr/lib/
```

---

oops smart